### PR TITLE
Added lots (see description)

### PR DESCRIPTION
**Pull request by ghost**
- [x] I fixed all the link tags to correct links
- [x] I changed the font to arial
- [x] I removed over-spacing in code
- [x] I used breakline tags to make website more readable
- [x] I fixed some grammar or punctuation mistakes
- [x] I added titles to each page

I would recommend adding a favicon or icon to your website.
<br>@GameDude734

### CODE REVIEW

**Comment by GameDude734** [comment]
thanks!

**Comment by ghost** [comment]
@GameDude734 
If you want to take my changes into effect, you have to hit the "Merge pull request" button.
You can see what code I changed by clicking the "Files Changed" tab.

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-meme.stencylxd.ga
```
Comment (ghost) [inline CNAME:1]: I removed it so I could test the website easily. I put it back in last commit.

**Comment by ghost** [comment]
Closed due to user inactivity.

**Pull request closed without merging**